postcss.config.cjs

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Do we need to have a PostCSS config now is because `react-script` use to do this automatically for us?

Reply:
@WiNloSt yes, I think react-scripts has a built-in PostCSS config, while Vite requires an explicit PostCSS configuration